src/App.js

```diff
@@ -1,18 +1,27 @@
 import './App.css';
-import initialCards from './cards-data';
 import Player from './components/Player';
 import CardList from './components/CardList';
-import { useState } from 'react';
+import { useContext, useState } from 'react';
 import ExecutePassButton from './components/ExecutePassButton';
+import initialCards from './cards-data';
+
+import { UserContext } from './context/GameContext.js';
 
 function App() {
-  const [deck, setDeck] = useState(initialCards);
-  const [playerOneHand, setPlayerOneHand] = useState([]);
-  const [selectedCard, setSelectedCard] = useState();
-  const [playerTwoHand, setPlayerTwoHand] = useState([]);
-  const [playerThreeHand, setPlayerThreeHand] = useState([]);
-  const [from, setFrom] = useState('deck');
-  const [to, setTo] = useState(1);
+  const {
+    to,
+    from,
+    selectedCard,
+    setSelectedCard,
+    playerOneHand,
+    setPlayerOneHand,
+    playerTwoHand,
+    setPlayerTwoHand,
+    playerThreeHand,
+    setPlayerThreeHand,
+    deck,
+    setDeck,
+  } = useContext(UserContext);
 
   function findCardIndex(value, suit, cards) {
```

Comment:
Notice how these functions are not being used here? The only reason they were living in `app` instead of `ExecutePassFunction` was because of state limitations that have been fixed by putting the state in context. Not docking any points for this, but try to see if you can move them out of here and into where they are being used. One less prop to pass, a cleaner-looking `app.js`.